vignettes/BiocParallel_BatchtoolsParam.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "Introduction to *BatchtoolsParam*"
+author:
+- name: "Nitesh Turaga"
+  email: Nitesh.Turaga@RoswellPark.org
+- name: "Martin Morgan"
+  email: Martin.Morgan@RoswellPark.org
+date: "Edited: March 22, 2018; Compiled: `r format(Sys.time(), '%B %d, %Y')`"
+vignette: >
+  %\VignetteIndexEntry{2. Introduction to BatchtoolsParam}
+  %\VignetteKeywords{parallel, Infrastructure}
+  %\VignettePackage{BiocParallel}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+    number_sections: true
+    toc: yes
+    toc_depth: 4
+---
+ 
+# Introduction
+
+The class is an interface to `batchtools` the package from within `BiocParallel`. This aims to
+replace BatchjobsParam as `BiocParallel`'s class for computing on a high performance
+cluster such as SGE, TORQUE, LSF, SLURM, OpenLava.
```

Comment:
I'm noticing that the text in the original vignette is slightly different from what is here. To make them consistent, please update this text to: 

The `BatchtoolsParam` class is an interface to the *[batchtools](https://cran.r-project.org/web/packages/batchtools/index.html)* package from within *[BiocParallel](http://bioconductor.org/packages/release/bioc/html/BiocParallel.html)*. This aims to replace BatchjobsParam as *[BiocParallel](http://bioconductor.org/packages/release/bioc/html/BiocParallel.html)’s* class for computing on a high performance cluster such as SGE, TORQUE, LSF, SLURM, OpenLava.